core/stse_frame.c

```diff
@@ -298,7 +299,7 @@ stse_ReturnCode_t stse_frame_receive(stse_Handler_t* pSTSE, stse_frame_t* pFrame
     while ((retry_count != 0) && (ret == STSE_PLATFORM_BUS_ACK_ERROR))
     {
 		/* - Receive frame length from target STSAFE */
-		ret = pSTSE->io.BusRecvStart(
+		ret = (stse_ReturnCode_t)pSTSE->io.BusRecvStart(
```

Comment:
Instead of casting it would be a better approach to change the return value type of `io.BusRecvStart` from **PLAT_UI16** to **stse_ReturnCode_t**. All the platforms are specified to return **stse_ReturnCode_t** so no constraint on this side.